index.py

```diff
@@ -5,17 +5,17 @@
 
 from colorama import Fore, Style
 
-# Make sure that the user is running Python 3.7 or higher
-if sys.version_info < (3, 7):
-    exit("Python 3.7 or higher is required to run this bot.")
+# Make sure that the user is running Python 3.8 or higher
+if sys.version_info < (3, 8):
+    exit("Python 3.9 or higher is required to run this bot!")
```

Comment:
Should probably say 3.8 instead of 3.9 lol

Reply:
How'd did that slip past me lmao